packages/core/admin/admin/src/pages/MarketplacePage/components/SortFilter/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { SimpleMenu, MenuItem } from '@strapi/design-system/SimpleMenu';
+import { useIntl } from 'react-intl';
+import { NavLink } from 'react-router-dom';
+import { getTrad } from '../../../../content-manager/utils';
+
+const SortFilter = () => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <SimpleMenu
+      variant="tertiary"
+      label={formatMessage({
+        id: getTrad('sort.label'),
+        defaultMessage: 'Sort by',
+      })}
+    >
+      <MenuItem as={NavLink} to="/marketplace?sort=name:asc">
```

Comment:
It's not very clear from the designs but there is something weird with the selected sort and the one being hovered: https://www.loom.com/share/c3dc45207b554e85a6167a9ccbab7abe

I think some margin is needed between the two, especially if it has the active border

Reply:
Also it doesn't seem to keep track of what the active sort is:
https://www.loom.com/share/4548b577f0bb4903a09e94b16b25dde2

When I select alphabetical close the menu and re-open I see alphabetical highlighted.
When I select newest close the menu and re-open I still see alphabetical highlighted even though it is being sorted by newest.